x-pack/plugins/enterprise_search/public/applications/shared/result/result.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+
+import {
+  EuiButtonIcon,
+  EuiCheckbox,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiToolTip,
+} from '@elastic/eui';
+
+// @ts-ignore
```

Comment:
Try without this, if possible

Reply:
What is the ts error this is generating? Because yeah: we should definitely not be ts-ignoring anything if at all possible.